streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -677,8 +677,6 @@ public void closeClean() {
         };
 
         resetToStrict(changeLogReader);
-        changeLogReader.transitToRestoreActive();
-        expectLastCall();
```

Comment:
This is a strict mock, so we have to remove this expectation, since the TaskManager no longer calls it.